business/_posts/2020-09-10-using-feature-dependency-graphs-for-software-planning.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Feature Dependency Graphs for Software Planning
+layout: post
+---
+
+When developing software systems there is a very common need among all of the
+stakeholders involved: to know *how we get from where we are today to some
+particular result*. In software product management, the result would be a
+“deliverable” or a “feature” that provides real, tangible value to customers or
+clients.
+
+## The Problem: An Engineering/Non-Engineering Gap
```

Comment:
I think this could also be described as a doer/decider gap or a doer/other-doer gap. Which is to say that we found there was a gap between design and engineering as well and design and product. Probably much more org dependent, but FDGs could also close a backend/mobile gap as well. I don't know if this is too confusing to try to get into at this early stage of the article or not…